packages/host/app/services/billing-service.ts

```diff
@@ -36,8 +49,39 @@ export default class BillingService extends Service {
 
   resetState() {
     this._subscriptionData = null;
+    this._paymentLinks = null;
+  }
+
+  managePlan() {
+    window.open(environment.stripeCustomerPortalLink);
+  }
+
+  async fetchPaymentLinks() {
+    if (!this._paymentLinks) {
+      await this.fetchPaymentLinksTask.perform();
+    }
+
+    return this._paymentLinks;
+  }
+
+  get paymentLinks() {
+    return this._paymentLinks;
   }
 
+  private fetchPaymentLinksTask = dropTask(async () => {
+    let response = await stripe.paymentLinks.list();
```

Comment:
I suppose you need the API key to fetch that? Having the API key in the front-end where anyone can see it is a giant security issue